r/Dockerfile

```diff
@@ -95,6 +95,8 @@ COPY CONDARC ./.condarc
 
 ENV PATH /usr/lib/rstudio-server/bin/:${PATH}
 
+ENV LD_LIBRARY_PATH=${CONDA_DIR}/lib:${LD_LIBRARY_PATH}
```

Comment:
@abkfenris very hack-y and untested. This is the same `start` file that fixes the issue when using repo2docker. Hopefully we will get the conda libstdc++ loaded first.